electron/libelectron/src/backends/memory_mgmt.cc

```diff
@@ -0,0 +1,115 @@
+#include "memory_mgmt.hh"
+
+namespace memory_mgmt {
+
+FirstFitAllocator::FirstFitAllocator(size_t dev_mem_size, void* device_memory) {
+
+    /* Initialize structures*/
+    heap_start = std::make_shared<MemBlock>();
+    heap_start->size = dev_mem_size;
+    heap_start->used = 0;
+    heap_start->ptr = device_memory;
```

Comment:
Please remove `device_memory` entirely from the allocator

Reply:
Start at address 0x0

Reply:
Done